src/serialization/sb3.js

```diff
@@ -434,33 +503,33 @@ const serializeComments = function (comments) {
 /**
  * Serialize the given target. Only serialize properties that are necessary
  * for saving and loading this target.
- * @param {object} target The target to be serialized.
+ * @param {RenderedTarget} target The target to be serialized.
+ * @param {Runtime} runtime The VM runtime
  * @param {Set} extensions A set of extensions to add extension IDs to
```

Comment:
I guess we have the same problem here.. so you can probably ignore the comment above.

Alternatively we can make it more descriptive -- the set of extension IDs from the blocks being used by this target (or some better phrasing)